### DISCUSSION TITLE
Using rustdesk android version on Meta Quest 3...

### DISCUSSION THREAD

**Discussion by chinanoahli** [rustdesk/rustdesk#7006] [Q&A]
I know, I know. Using rustdesk on VR devices is not one of your team's priorities.
So I posted the issue when I tried to use RustDesk on Quest3 on discussions, but not issues.

So, here is the problem I got, when I'm using Quest3 to control my PC on LAN only:
1. No matter using mouse mode or touch screen mode, I cannot do any clicks. I mean, the cursor can move follow the Quest3 controller's point, but cannot recognize the click action.
2. If I move my head too fast, rustdesk will crash the Quest3's system, and I have to reboot my Quest3.

**Comment by jurassicjordan** [comment]
I'm having the same problem with the quest 2, I'd love to stay foss with a screen casting solution
